partner_contact_address_default/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    allow_all_partner_contact_address = fields.Boolean()
```

Comment:
To make the correlation to the module a bit clearer.
```suggestion
    contact_address_default_allow_all_partners = fields.Boolean()
```